tests/unit/date/core.js

```diff
@@ -1,19 +1,21 @@
 define( [
 	"jquery",
-	"globalize",
+	"./helper",
 	"date"
-], function( $, Globalize ) {
+], function( $, testHelper ) {
 
 module( "date: core" );
 
+var attributes = testHelper.getAttributes( "en" );
+
 test( "Instantiation", function() {
 	expect( 2 );
-	ok( new $.date() instanceof $.date, "constructor function" );
-	ok( $.date() instanceof $.date, "instantiation without new" );
+	ok( new $.ui.calendarDate( null, attributes ) instanceof $.ui.calendarDate, "constructor function" );
```

Comment:
I still don't really understand the scope of this object. Is it useful on its own? Is it intended to be used by others?

One thing that's clear to me though is that it cannot simultaneously be in `/external/` and use the `$.ui` namespace.